jekyll/_config.yml

```diff
@@ -21,6 +21,11 @@ plugins:
   - jekyll-target-blank
   - jekyll-toc
   - jekyll/minibundle
+  - jekyll-last-modified-at
+
+# # Optional. The default date format, used if none is specified in the tag.
+# last-modified-at:
+#     date-format: '%d-%b-%y'
```

Comment:
what is b?

Reply:
b is month

Reply:
https://www.ibm.com/docs/en/cmofm/9.0.0?topic=SSEPCD_9.0.0/com.ibm.ondemand.mp.doc/arsa0257.htm
<img width="819" alt="Screen Shot 2021-12-16 at 3 05 11 PM" src="https://user-images.githubusercontent.com/86666932/146441581-4dfaa34f-34c4-4ed7-ab2e-35afab85e91e.png">

It will say what the month is but short form like Jan, Feb, Mar

Reply:
weird. maybe add that in comment?

Reply:
can do!